marathon.json

```diff
@@ -7,13 +7,13 @@
   "container": {
     "type": "DOCKER",
     "docker": {
-      "image": "thomasr/dcos-openvpn:25",
+      "image": "mesosphere/dcos-openvpn",
```

Comment:
So the force-pull for the server container is already in master. I've just pushed another commit to force-pull the image for the framework as well.